etc/taskcluster/macos/states/generic-worker.sls

```diff
@@ -49,36 +56,27 @@ GMT:
         workerId: {{ grains["id"] }}
         tasksDir: {{ home }}/tasks
         publicIP: {{ salt.network.ip_addrs()[0] }}
-        signingKeyLocation: {{ home }}/key
+        ed25519SigningKeyLocation: {{ home }}/keypair
         clientId: {{ pillar["client_id"] }}
         accessToken: {{ pillar["access_token"] }}
+        taskclusterProxyExecutable: {{ bin }}/taskcluster-proxy
+        taskclusterProxyPort: 8080
         livelogExecutable: {{ bin }}/livelog
-        livelogCertificate: {{ etc }}/livelog.crt
-        livelogKey: {{ etc }}/livelog.key
-        livelogSecret: {{ pillar["livelog_secret"] }}
+        wstAudience: taskcluster-net
+        wstServerURL: https://websocktunnel.tasks.build
+        rootURL: https://taskcluster.net
     - watch_in:
       - service: net.generic.worker
 
-{{ etc }}/livelog.crt:
-  file.managed:
-    - contents_pillar: livelog_cert
-    - group: {{ user }}
-    - mode: 640
-
-{{ etc }}/livelog.key:
-  file.managed:
-    - contents_pillar: livelog_key
-    - group: {{ user }}
-    - mode: 640
-
-{{ bin }}/generic-worker new-openpgp-keypair --file {{ home }}/key:
+{{ bin }}/generic-worker new-ed25519-keypair --file {{ home }}/keypair:
   cmd.run:
-    - creates: {{ home }}/key
+    - creates: {{ home }}/keypair
     - runas: {{ user }}
 
 /Library/LaunchAgents/net.generic.worker.plist:
   file.managed:
-    - mode: 644
+    - mode: 600
+    - user: root
```

Comment:
Is it intended to run tasks as root?

Reply:
No. As far as I understand the service runs as the user [specified inside of the `.plist` file](https://github.com/servo/servo/blob/75b67a63e69a7ed243ea2c734f088436fbe23a99/etc/taskcluster/macos/states/generic-worker.plist.jinja#L10), not the one that owns it.

`launchctl` was complaining and https://stackoverflow.com/questions/28063598/error-while-executing-plist-file-path-had-bad-ownership-permissions suggested this fix.